examples/microphone_nextjs/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "speechmatics-mic-demo",
+  "version": "0.1.0",
+  "scripts": {
+    "dev": "next dev --port 80",
+    "build": "next build",
+    "start": "node server.js",
```

Comment:
`server.js` does not currently exist, so this will fail

Reply:
Ah yeah, I removed server.js to keep this example app a bit more minimal forgot to delete the command